integrations/elasticsearch/integration.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+import json
+import requests
+
+from integrationmanager.exceptions import IntegrationSendEventError
+from integrationmanager.integration_utils import BaseIntegration
+
+logger = logging.getLogger(__name__)
+
+
+class ElasticsearchIntegration(BaseIntegration):
+
+    def send_event(self, alert_fields):
+        address = self.integration_data["url_address"]
+        port = self.integration_data["port"]
+        index = self.integration_data["index"]
+        user = self.integration_data.get("username", "").encode('utf-8')
+        passwd = self.integration_data.get("password", "").encode('utf-8')
+        url = "{}:{}/{}".format(address, port, index)
+        skip_cert_validation = self.integration_data.get('skip_cert_validation', False)
+        format_alert = json.dumps(alert_fields,
+                                  indent=4,
+                                  sort_keys=True,
+                                  default=str)
+        response = requests.post(url=url,
```

Comment:
Create a session so you'll enjoy keep alive, etc. 
http://docs.python-requests.org/en/master/user/advanced/#session-objects